packages/genesys-spark-components/.stylelintrc.json

```diff
@@ -1,15 +1,7 @@
 {
   "extends": ["stylelint-config-genesys-spark-components"],
-  "overrides": [
-    {
-      "files": [
-        "src/components/stable/gux-button/**/*.{scss,html}",
-        "src/components/stable/gux-button-slot/**/*.{scss,html}"
-      ],
-      "plugins": ["stylelint-use-logical-spec"],
-      "rules": {
-        "liberty/use-logical-spec": true
-      }
-    }
-  ]
+  "plugins": ["stylelint-use-logical-spec"],
+  "rules": {
+    "liberty/use-logical-spec": true
+  }
```

Comment:
Looking at this PR again I think we should do this in a few PRs over a few weeks to reduce the risk. We are also not going to want to change the legacy components at all so we will need to have an override for that.

See can you find a config that changes about 40 files. Also have the example.html files changes in their own PR as they are not that critical.

Reply:
Was thinking it might be good to start with beta components and go from there.